doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -420,6 +420,9 @@ Modem libraries
       * Obstructed satellite visibility detection feature for GNSS.
         When this feature is enabled, the library tries to detect occurrences where getting a GNSS fix is unlikely or would consume a lot of energy.
         When such an occurrence is detected, GNSS is stopped without waiting for a fix or a timeout.
+      * In addition to existing default fallback mode for acquiring a location,
+        now on location can be acquired by using mode LOCATION_REQ_MODE_ALL that runs
+        all methods sequentially in methods list. Each run methods will receive a location event, either a success or a failure.
```

Comment:
```suggestion
      * In addition to the current default fallback mode for acquiring a location, it can also be acquired using the :enumerator:`LOCATION_REQ_MODE_ALL` mode that runs all methods in the list sequentially.
        Each run method receives a location event, either a success or a failure.
```
Not that sure whether LOCATION_REQ_MODE_ALL is an enumerator or something else. Please check https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#breathe and use the appropriate tag if I got it wrong.

Reply:
Yes, it is an enum. Changed as was suggested. Thanks!